### PR TITLE
fix: add a11y jsx linter

### PR DESCRIPTION
**Pull request by sabrinaheid**
## Preflight Checklist

* [ ] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/master/.github/CONTRIBUTING.md) for this project.
* [ ] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/master/.github/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Changes

Changes in this pull request:

-

## Browsers

I tested the build on the following browsers:
- [ ] Firefox Desktop
- [ ] Chrome Desktop
- [ ] Safari Desktop
- [ ] Firefox Mobile
- [ ] Chrome Mobile
- [ ] Safari Mobile
- [ ] Opera
- [ ] Brave

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Mentions

@lyne-design-system/core

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,10 +27,11 @@
     "chromatic": "chromatic --storybook-build-dir=./storybook-static --exit-zero-on-changes",
     "chromatic:master": "chromatic --storybook-build-dir=./storybook-static --exit-zero-on-changes --auto-accept-changes",
     "generate": "stencil generate",
-    "lint": "npm run lint:ts && npm run lint:js && npm run lint:yml",
+    "lint": "npm run lint:ts && npm run lint:js && npm run lint:yml && npm run lint:a11y",
```
Comment (feerglas) [inline package.json:30]: also `npm run lint:a11y` lintet ja `.tsx` und `.jsx`. `.jsx `werden wir wahrscheinlich nie haben, da wir sowieso alles mit typescript machen. Von daher kannst du `... && npm run lint:a11y` weglassen. Somit wird dann das accessibility linting von `lint:ts` gemacht. Wenn wir es so lassen würden, dann würde das linting für tsx files ja sowieso immer doppelt ausgeführt.

Comment (sabrinaheid) [reply package.json:30]: stimmt das ist ja doppelt, weiss nicht was ich hier überlegt habe

Comment (sabrinaheid) [reply package.json]: stimmt das ist ja doppelt, weiss nicht was ich hier überlegt habe

**Pull request merged**